script/test_todo.py

```diff
@@ -0,0 +1,45 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import NoSuchAttributeException
+
+
+def main():
```

Comment:
Let's start fixing this code by some rafactoring:
Try to refactor function main into a list of functions which would resemble the list of steps from the issue.

For example:
  * Add "task1" and press enter

The code for this should looks like this:
```python
def add_task(driver, title):
   input_todo = driver.find_element_by_xpath("//html/body/section/div/header/input")
   input_todo.clear()
   input_todo.send_keys("task1")
   input_todo.send_keys(Keys.RETURN)


def main():
   driver = ...
   add_task(driver, "task1")
  # Rest of the actions
```

Create actions for all steps and use them in function main.

Reply:
Ok :) I was just wondering how to solve the problem of repetitive code. Now I know .